docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
-the Tezos network. This integration and the interactions that come with it are
-delegated to a general-purpose node called the Smart Rollup node, which listens
-and reacts to what is happening on the Tezos network by being connected to an
-`octez-node`.
+Anyone can run an Etherlink node to join the network, but operating an Etherlink node requires some familiarity with the Tezos ecosystem, Smart Rollups, and the Octez suite.
+
+## Transaction life cycle
+
+To run an Etherlink node, you must understand how transactions flow through Etherlink.
+
+In its life cycle, an Etherlink transaction goes through three distinct types of nodes, each responsible for different aspects of Etherlink's features.
+These three nodes are being developed as part of the [Octez software suite](https://tezos.gitlab.io/introduction/tezos.html):
+
+- Etherlink nodes (`octez-evm-node`): The Etherlink nodes themselves maintain a local copy of the Etherlink context, apply new blocks as they are created, and expose a [JSON RPC API](https://ethereum.org/en/developers/docs/apis/json-rpc/) compliant endpoint to clients.
+- Smart Rollup nodes (`octez-smart-rollup-node`): Etherlink nodes delegate interactions with Tezos layer 1 nodes through these Smart Rollup nodes.
+- Tezos layer 1 nodes, also referred to as Octez nodes (`octez-node`): The Smart Rollup nodes listen to Tezos layer 1 nodes and react to what happens on the Tezos network.
 
 The overall lifecycle of a typical operation is as follows:
 
-1. One user submits their transaction to an `octez-evm-node` using the
-   `eth_sendRawTransaction`.
-2. This transaction is eventually forwarded to Etherlink’s sequencer, which
-   includes it in a new block as soon as possible (less than 500ms after
-   receiving it in a nominal scenario).
-3. The sequencer forwards this new block to the Smart Rollup node it is
-   connected to, which is tasked to post this data on the Tezos network.
-   Besides, the sequencer also makes the new block available to other
-   `octez-evm-node` it is connected to, which can do the same recursively.
-4. The Smart Rollup nodes tracking the state of Etherlink fetch this new
-   Etherlink block and validate it.
-5. This allows the `octez-evm-node`s to assert the block they received
-   previously has indeed be faithfully posted by the sequencer.
-6. Eventually, the Smart Rollup nodes assuming the role of “operators”
-   commit the state of Etherlink to the Tezos network.
-
-We can summarize the connections between each component with this diagram.
+1. A user submits a transaction to an Etherlink node.
+2. The Etherlink node forwards the transaction to Etherlink's sequencer, which includes it in a new block as soon as possible (less than 500ms after receiving it in a nominal scenario).
+3. The sequencer forwards the new block to the Smart Rollup node it is connected to.
+4. The sequencer also shares the new block available with other Etherlink nodes, which continue to share it in a peer-to-peer manner.
```

Comment:
you lost the part where the rollup node must submit the block to the rollup inbox in the l1.

Reply:
@sribst Based on discussion with @jobjo, the sequencer sends the block directly to layer 1. Do I have something wrong? Have a look at the new version and diagram.

Reply:
No, did not mean to imply that. The sequencer posts blueprints through the rollup node.  It also offers other evm nodes to subsribe to the blueprints. There is no peer-to-peer network involved for propagating blueprints however.

Reply:
Ok, I got that wrong. How about this?
https://github.com/etherlinkcom/docs/pull/49/commits/9ff0a7d33c58798107491df3a55317b9f40a8115